backend/src/services/password-validation/password-validation.service.ts

```diff
@@ -0,0 +1,11 @@
+import { User_Security } from '@prisma/client';
+import { compare as bcryptCompare } from 'bcrypt';
+
+const validatePassword = async (
+  password: string,
+  userSecurity: User_Security | null | undefined,
+): Promise<boolean> => {
+  return bcryptCompare(password, userSecurity?.password || '');
+};
```

Comment:
Curly braces are redundant for the one-line funcs:
```suggestion
const validatePassword = async (
  password: string,
  userSecurity: User_Security | null | undefined,
): Promise<boolean> =>
  bcryptCompare(password, userSecurity?.password || '');
```